src-vue/src/views/mods/LocalModsView.vue

```diff
@@ -0,0 +1,122 @@
+<template>
```

Comment:
Is there a reason this source file is in a different folder? ^^

`views/` vs `mods/`

Reply:
Oops, I forgot to put TS view into this folder as well 😄 
Addressed in 9fd76811f9c938fc5e12f4bc9162ad7890eef90f